src/system/SearchModule/Block/SearchBlock.php

```diff
@@ -11,7 +11,6 @@
 
 namespace Zikula\SearchModule\Block;
 
-use ModUtil;
```

Comment:
> do not import from global namespace

does this make a difference?

Personally I prefer the import against using `\Foo` because it is more explicite about which classes are used within a class. Just a matter of taste or is there a behavioural difference?